tagreader.yaml

```diff
@@ -17,8 +17,11 @@ esphome:
   name: $devicename
   platform: ESP8266
   board: d1_mini
-  includes: 
-    - rfidreader/src/test.h
+  includes:
+  - includes/ndef.h
+  libraries:
+  - https://github.com/jesserockz/NDEF.git
```

Comment:
We should try to upstream the fixes so we in the future no longer need to rely on forks

Reply:
The "fixes" are stripping out the files that cause ESPHome compile warnings and errors... 
Probably not something that can be upstreamed... 
Given time I may be able to integrate it directly into esphome. 
This is more of a quick win to get it out before they are all sent out pre-flashed.